util/regexp-assemble/data/942130.data

```diff
@@ -23,19 +23,54 @@
 
 ##!+ i
```

Comment:
I'd move the general comments before the flags. From the perspective of the regex, the flags are the start of the expression, so I intuitively would expect general information to precede them.

Reply:
Moving.